### PR TITLE
allow creating secrets from secret converter

### PR DESCRIPTION
**Pull request by yuval-k**
once https://github.com/solo-io/solo-kit/pull/110 is merged we can do some minor cleanup

### CODE REVIEW

File: projects/gloo/pkg/api/converters/kube/secretconverter_test.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ var _ = Describe("Secretconverter", func() {
 		Expect(glooSecret.PrivateKey).To(BeEquivalentTo(secret.Data[kubev1.TLSPrivateKeyKey]))
 	})
 
-	It("should NOT covnert to gloo secret kube secret", func() {
+	It("should covnert to gloo secret kube in gloo format", func() {
```
Comment (rickducott) [inline projects/gloo/pkg/api/converters/kube/secretconverter_test.go]: convert

**Comment by yuval-k** [comment]
/skip-changelog as this has no effect on gloo

**Comment by rickducott** [comment]
knative webhook flake #545

**Comment by rickducott** [comment]
I attempted to fix this issue here: #581

**Pull request merged**